src/BlockNode/types/BlockNodeConstructorParameters.ts

```diff
@@ -0,0 +1,27 @@
+import { TextNode } from '../../TextNode';
+import { ValueNode } from '../../ValueNode';
+import { FormattingNode } from '../../FormattingNode';
+import { EditorDocument } from '../../EditorDocument';
+import { BlockTune } from '../../BlockTune';
+
+export interface BlockNodeConstructorParameters {
+  /**
+   * The name of the BlockNode
+   */
+  name: string;
+
+  /**
+   * The child nodes of the BlockNode
+   */
+  children: Record<string, TextNode | ValueNode | FormattingNode>;
+
+  /**
+   * The parent EditorDocument of the BlockNode
+   */
+  parent: EditorDocument;
+
+  /**
+   * The BlockTunes associated with the BlockNode
+   */
+  tunes: Record<string, BlockTune>;
```

Comment:
tunes?
```suggestion
  tunes?: Record<string, BlockTune>;
```

Reply:
I think it would be better to create an empty object as the default value. What do you think?

Reply:
Oh, it was a comment in a constructor params type. Yep, I agree with you. Fixed.